.travis.yml

```diff
@@ -25,12 +25,16 @@ branches:
     - /^amp-release-.*$/
     - /^revert-.*$/
 addons:
+  apt:
+    packages:
+      - protobuf-compiler
+      - python-protobuf
+  chrome: stable
```

Comment:
nit: Same here, only the Local Tests and Dist/Bundle Size/Single Pass jobs need Chrome but it takes ~1 second.

Reply:
Same reason as above.